### PR TITLE
Sixth milestone: Evaluate accessibility

### PR DESCRIPTION
**Pull request by AkashaRojee**
Hello!

Together with my coding partner @NJ-2020, we evaluated and corrected the accessibility of the portfolio.

Things done:

- Check accessibility for page titles, image text alternatives, text headings, color contrast, resize, interaction, moving content, multimedia and the basic structure of the page
- Report on issues found
- Correct issues found

Please, [view the report](https://docs.google.com/document/d/1J1-dR5HLjIal3EseDAhP7U2e9xTkMy1rRr-ZDxWA0ck/edit?usp=sharing) for more information and review the code.

Thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,15 +23,14 @@
 
       gtag("config", "G-L9EM4V5T72");
     </script>
->>>>>>> Application-deploy
-    <title>Application Deploy</title>
+    <title>My Portfolio</title>
```
Comment (Temmarie) [inline index.html]: - [x] Kindly fix this title to reflect what the site is about.

File: index.html [hunk 2]
```diff
@@ -25,15 +23,14 @@
 
       gtag("config", "G-L9EM4V5T72");
     </script>
->>>>>>> Application-deploy
```
Comment (Temmarie) [inline index.html:28]: - [x] There's a weird bar at the top of the page, kindly remove it. It is from a git merge.
image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @AkashaRojee and @NJ-2020     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

- - -

### **Required Changes** ♻️
**Your report has been analysed, see the feedback below and make the changes.**
- [x] Your report was spot on! Good job there. However, according to the project requirements, you are to evaluate your website against accessibility best practices and implement any necessary changes to ensure that your website is accessible. You have currently only pointed out the issues, kindly implement the necessary changes to ensure accessibility.
- [x] As mentioned in the project requirements, If you need to change something, link to the corresponding line of code in your Pull Request changes. 👼🏾 
You've gone the extra mile by creating a google doc and putting the report there but kindly copy and paste the report in the PR description message, including the links to the corresponding line of code.


**Your report has been analysed, see the feedback below and make the changes.**
 **Check** | **Feedback**
  ----------- | -----------
 | Page Title  | It should adequately and briefly describe the content of the page.
 |                   | Poor title:  My Portfolio Better title: Moses' Portfolio. |
 | Headings   | Good job updating the headings, however, a page should only have one `h1` tag.  Since you have more than one `h1` tag on your page, change the other `h1` tags to an `h2` or `h3`. Your `h1` tag should be at the top of the page content (above any other heading tags in the page code). |
| Colour contrast | See comment below |

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- The `submit` button on the form has a typographical error. Kindly fix this. It should be `Get In Touch` not `Get It Touch`.

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Comment by AkashaRojee** [comment]
Hello!

Thanks for the feedback.

The changes had already been implemented but the deploy was happening from another branch. The deploy branch has now been corrected.

Additional changes made:

- Improve page title
- Improve document outline
-- Change Contact Me from h1 to h2
-- Change h2 headings of individual work cards to h3
-- Add hidden h2 heading for Works section
- Correct typo on the contact form submit button

Please, view the accessibility report below. Thanks!

# 1. Page titles

## Issue: Title does not adequately describe page content

### Corrections:
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L26

# 2. Image text alternatives

## Issues:
* Images have empty alt attributes (checked with WebDev Toolbar)
* Images do not have adequate alt attributes

### Corrections:
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L33
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L63
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L66
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L69
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L72
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L75
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L86
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L111
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L138
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L165
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L207
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L212
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L217
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L222
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L227
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L239
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L242-L244
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L251
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L259

# 3. Text headings (checked with WebDev Toolbar)

## Issue: Let’s Connect should not be a heading but can be a paragraph

### Correction:
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L59
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L203

## Issue: Logo should be an image and not a heading

### Correction:
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L33

## Issue: Contact Me should be h2 instead h1

### Correction
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L269

## Issue: Headings of individual work containers should be h3 instead of h2

### Correction: 
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L82
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L99-L101
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L89
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L114
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L141
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L168

# 4. Colour contrast

## Issues:
* Some elements do not have sufficient color contrast (checked with WCAG Color Contrast Checker for Chrome) because the colours used in the Figma template itself lack sufficient color contrast.
* When viewing in grayscale (checked with Grayscale the Web for Chrome), some elements are difficult to read because of the same reason above.

### Corrections:
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L71
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L143
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L170
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L179
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L182
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L414
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L426
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L428
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L434
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L602
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L605

# 5. Resize (checked with Zoom Text Only for Chrome)

## Issue: Some elements become hidden behind others

### Corrections:
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L48

# 6. Interaction

## Issue: Buttons not showing tab focus

### Corrections:

Removed "outline: none" from CSS (no link available because code was removed to apply the correction)

## Issue: Let’s Connect is receiving tab focus but should not

### Corrections:
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L59
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/style.css#L70

## Issue: Arrows to collapse/expand Skills section do not receive tab because they are currently images

### Corrections:
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L239
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L251
https://github.com/NJ-2020/My-Portfolio-Web-Designs/blob/d765514ec227227bbd48687a1f4ad5b96457146d/index.html#L259

# 7. Moving content

N/A

# 8. Multimedia

N/A

# 9. Basic structure of page

No issue found (checked with WebDev Toolbar).

**Review by teekaytech — APPROVED** [review]
## Status: Approved :green_circle: 

### Congratulations team :tada: :clinking_glasses:

**You've made significant improvements compared to the first review :handshake: 
Your evaluation report is comprehensive and you've typically worked on all the required changes :tada: 
Your project is complete!
This is a convenient point to merge :rocket:**

- - - - - -

**_Feel free to leave any questions or comments in the PR thread if something is not 100% clear._**

Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
_Cheers,_
**[Taofeek Olalere](https://github.com/teekaytech)**
__Happy Coding :smiley:__

**Pull request merged**